omnigibson/action_primitives/starter_semantic_action_primitives.py

```diff
@@ -1003,7 +1035,8 @@ def _navigate_to_pose(self, pose_2d):
             self.robot.set_position_orientation(*robot_pose)
             yield from self._settle_robot()
         else:
-            with UndoableContext(self.robot, "base") as context:
+            robot_copy_type = "simplified" if self.robot_model == "Tiago" else "original"
```

Comment:
This is annoying to repeat. Let's put it inside the code for UndoableContext.

Reply:
If I remove that parameter for Tiago and include that logic in UndoableContext, Tiago will always use the simplified model and we don't want that for its arm movement. An option is I use the mode parameter that is "base" or "arm" like I did before and figure out the mesh copy type and and whether we want self collisions in UndoableContext. I removed that because I thought this would make it more clear as to what is happening in UndoableContext. Let me know which you want.